pymess/models/emails.py

```diff
@@ -277,6 +277,7 @@ class Meta(BaseRelatedObject.Meta):
 class EmailTemplateAttachment(SmartModel):
 
     template = models.ForeignKey(to=settings.EMAIL_TEMPLATE_MODEL, verbose_name=_('template'),
+                                 null=True, blank=True,
```

Comment:
this should probably stay null/blank False

Reply:
I added this to be consistent with all `Message` classes (e.g., `EmailMessage`). On the other hand, I get that for this model it doesn't make much sense. I was conflicted on whether to make the extra effort as the null/blank=True are needed for the migration but I can make them `False` again at the end of the migration.

What do you think @rubickcz

Reply:
This object makes no sense without reference to a template. So I agree with @Formulka.